server/database/db_build.js

```diff
@@ -1,13 +1,12 @@
 const fs = require('fs')
+
 const dbConnection = require('./db_connection')
+
 const sql = fs.readFileSync(`${__dirname}/db_build.sql`).toString()
-const runDbBuild = cb => dbConnection.query(sql, cb)
 
-runDbBuild((err, res) => {
+dbConnection.query(sql, (err, res) => {
   if (err) {
     throw err
   }
-  return res
+  console.log('Super heroes table created with result: ', res)
```

Comment:
Superheroes table?

Reply:
I think db_build.js is okay as that's how we've taught it.

Reply:
I changed it from runDbBuild to dbConnection.
 @FarahZaqout @jema28